tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -483,6 +483,10 @@
   </data>
   <data name="CloseButtonText" xml:space="preserve">
     <value>Close</value>
-    <comment>Text for a button to close a content dialog.</comment>
+    <comment>Text for a button to close a content dialog</comment>
+  </data>
+  <data name="MoreOptionsButton.AutomationProperties.Name" xml:space="preserve">
```

Comment:
I think you're supposed to use the `<data name="MyElementName.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">` syntax.
https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/mrtcore/localize-strings#refer-to-a-string-resource-identifier-from-xaml

Quickstart playground, where they do it the other way, probably has a bug.